### ISSUE TITLE
Enhancement: Add BattyBirdNet Bat Identification

### ISSUE THREAD

**Issue by PersephoneKarnstein** [tphakala/birdnet-go#319]
## Issue Type
- [  ] Bug report
- [x] Feature request
- [  ] Question / Support

## Description
I would love if you could add support for bat identification a la [BattyBirdNet](https://github.com/rdz-oss/BattyBirdNET-Analyzer)! BN-go is different enough from BN-pi (and I'm not a Go dev) that i'm not confident that it's as simple as dropping the modified BattyBirdNET [model](https://github.com/rdz-oss/BattyBirdNET-Pi/tree/main/model) into the [equivalent folder](https://github.com/tphakala/birdnet-go/tree/main/internal/birdnet/data)

## Additional context
Bats are cool

**Comment by tphakala** [comment]
Yes I have been thinking of this, and at some level this should already be somewhat possible with latest container dev images which allow to use custom classifier (BirdNET AI model).

**Comment by tphakala** [comment]
This is now implemented
